### PR TITLE
Create mod-arena-top-pvp

### PR DESCRIPTION
**Pull request by xDevICCI**

### CODE REVIEW

**Comment by milestorme** [comment]
Just need to make the change to cmake lists for arena stats I did a pr to your repo after testing the module on Debian .

**Comment by xDevICCI** [comment]
ok, thanks u bro <3

**Review by milestorme — APPROVED** [review]

File: _submissions/mod-arena-top-pvp [hunk 1]
```diff
@@ -0,0 +1,18 @@
+Module info
+Name: module_top_arena_pvp
+Author: Veridisquo
+Repository: https://github.com/xDevICCI/module_top_arena_pvp
+Download: https://github.com/azerothcore/module_top_arena_pvp/archive/master.zip
```
Comment (FrancescoBorzi) [inline _submissions/mod-arena-top-pvp]: shouldn't be the same repositories?

File: _submissions/mod-chat-censure [hunk 2]
```diff
@@ -0,0 +1,19 @@
+Name: Chat Censure
+Author: Veridisquo
+Repository: https://github.com/xDevICCI/module_chat_censure
+Download: https://github.com/poszer/module_chat_censure/archive/master.zip
```
Comment (FrancescoBorzi) [inline _submissions/mod-chat-censure]: same here

Comment (xDevICCI) [reply _submissions/mod-chat-censure]: changed, i used same models description of poszer

File: _submissions/mod-arena-top-pvp (hunk 1 above)
Comment (xDevICCI) [reply _submissions/mod-arena-top-pvp]: changed !

**Review by BarbzYHOOL — CHANGES_REQUESTED** [review]
Why is there a second file mod_chat_censure?

**Comment by milestorme** [comment]
> Why is there a second file mod_chat_censure?

He is submitting 2 modules, he just didn’t do 2 pr’s 😊. I may have not explained that part properly when him and I spoke on discord .

**Review by FrancescoBorzi — APPROVED** [review]

**Review by BarbzYHOOL — CHANGES_REQUESTED** [review]
Ok, thank you for these modules!


The repositories are not structured correctly, you must extract them at the root. Compare to the skeleton-module

also you should add the files from the skeleton-module like .gitattributes etc https://github.com/azerothcore/skeleton-module/

**Comment by milestorme** [comment]
> Ok, thank you for these modules!
> 
> The repositories are not structured correctly, you must extract them at the root. Compare to the skeleton-module
> 
> also you should add the files from the skeleton-module like .gitattributes etc https://github.com/azerothcore/skeleton-module/

@xDevICCI you can pm me in discord for assistance with that if you need

**Comment by xDevICCI** [comment]
> > Ok, thank you for these modules!
> > The repositories are not structured correctly, you must extract them at the root. Compare to the skeleton-module
> > also you should add the files from the skeleton-module like .gitattributes etc https://github.com/azerothcore/skeleton-module/
> 
> @xDevICCI you can pm me in discord for assistance with that if you need

ok ! thanks ! my nickname in discord is Veridisquo

**Comment by Helias** [comment]
What about the status of this PR?
I can't find "mod-arena-top-pvp" in your profile @xDevICCI, I've found 2 modules, why don't make a PR for them? :P

**Comment by milestorme** [comment]
@Helias He got these modules from Poszer I can fix them on my repo and do a fresh pull if you like , I tryed to get him to match our structure but he gave up

**Comment by Helias** [comment]
Ok, hence, I think we can close this PR and open a new one.

**Pull request closed without merging**